package/network/utils/curl/Makefile

```
@@ -64,7 +64,9 @@ PKG_CONFIG_DEPENDS:= \
   CONFIG_LIBCURL_TLS_SRP \
   CONFIG_LIBCURL_UNIX_SOCKETS \
   CONFIG_LIBCURL_VERBOSE \
-  CONFIG_LIBCURL_NTLM
+  CONFIG_LIBCURL_NTLM \
+  \
+  $(if $(CONFIG_LIBCURL_OPENSSL),CONFIG_OPENSSL_WITH_NPN,)
```

```suggestion
  $(if $(CONFIG_LIBCURL_OPENSSL), \
	CONFIG_OPENSSL_ENGINE \
	CONFIG_OPENSSL_WITH_COMPRESSION \
	CONFIG_OPENSSL_WITH_NPN \
	 \
	 \
)
```
Apparently, NPN is not the only configuration option being used by curl.  I've found at least these two in `lib/vtls/openssl.c`, and haven't checked what happens if `OPENSSL_WITH_EC` changes.